package.json

```diff
@@ -127,19 +135,19 @@
     "lint:fix": "npm run lint:js:fix",
     "lint:js": "eslint --ext .js,.jsx .",
     "lint:js:fix": "eslint --fix --ext .js,.jsx .",
-    "lint:scss": "stylelint 'packages/**/src/**/*.scss' --syntax scss",
+    "lint:scss": "stylelint 'packages/**/*.scss' --syntax scss",
```

Comment:
Will this lint scss in the lib folder now too?

Reply:
It shouldn't now that there's a `.stylelintignore`